### ISSUE TITLE
micronaut-function-aws-lambda graalvm native-image build fails

### ISSUE THREAD

**Issue by samblake** [micronaut-guides/micronaut-function-aws-lambda#12]
I am unable to build a native application using the [micronaut-function-aws-lambda](https://guides.micronaut.io/micronaut-function-aws-lambda/guide/index.html) example. However, I am able to create a native application following [micronaut-creating-first-graal-app](https://guides.micronaut.io/micronaut-creating-first-graal-app/guide/index.html). 

### Steps to Reproduce

1. git clone https://github.com/micronaut-guides/micronaut-function-aws-lambda.git
2. cd micronaut-function-aws-lambda/complete/vies-vat-validator-graal/
3. sdk use java 19.2.1-grl
4. gu install native-image
5. ./gradlew assemble
6. native-image --no-server --class-path build/libs/vies-vat-validator-0.1-all.jar

When run with `native-image --no-server --no-fallback --class-path build/libs/vies-vat-validator-0.1-all.jar` fails with:

```
Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of org.apache.logging.slf4j.Log4jLogger are allowed in the image heap as this class should be initialized at image runtime. To see how this object got instantiated use -H:+TraceClassInitialization.
```

Expected Behaviour
Native application should be built

Actual Behaviour
Build Fails

Environment Information
Operating System: Manjaro 18.1.2
Micronaut Version: 1.2.7
JDK Version: OpenJDK 64-Bit GraalVM CE 19.2.1 (build 25.232-b07-jvmci-19.2-b03, mixed mode)
Example Application
https://github.com/micronaut-guides/micronaut-function-aws-lambda.git

**Comment by samblake** [comment]
Fix is to add `--initialize-at-build-time=org.apache.logging.log4j,org.apache.logging.slf4j`

**Comment by ilopmar** [comment]
Fixed in #11
